integration-tests/actions/actions.go

```diff
@@ -474,36 +476,12 @@ func GenerateWallet() (common.Address, error) {
 	return crypto.PubkeyToAddress(*publicKeyECDSA), nil
 }
 
-// todo - move to CTF
-func FundAddress(client blockchain.EVMClient, sendingKey string, fundingToSendEth *big.Float) error {
-	address := common.HexToAddress(sendingKey)
-	gasEstimates, err := client.EstimateGas(ethereum.CallMsg{
-		To: &address,
-	})
-	if err != nil {
-		return err
-	}
-	err = client.Fund(sendingKey, fundingToSendEth, gasEstimates)
-	if err != nil {
-		return err
-	}
-	return nil
-}
-
 // todo - move to CTF
 func GetTxFromAddress(tx *types.Transaction) (string, error) {
 	from, err := types.Sender(types.LatestSignerForChainID(tx.ChainId()), tx)
 	return from.String(), err
 }
 
-// todo - move to CTF
-func GetTxByHash(ctx context.Context, client blockchain.EVMClient, hash common.Hash) (*types.Transaction, bool, error) {
-	return client.(*blockchain.EthereumMultinodeClient).
-		DefaultClient.(*blockchain.EthereumClient).
-		Client.
-		TransactionByHash(ctx, hash)
-}
-
```

Comment:
please restore these actions, as I said from the beginning: **we should not be deleting any of them**. Are you 100% sure CCIP doesn't need them?

Reply:
@Tofel yes, I'm sure I reviewed CCIP code. If there will be some code we miss we can always add it during CCIP - core migration